### PR TITLE
Feat: Improvements

### PR DESCRIPTION
**Pull request by Rohitprasad83**
- Changed Name from Games-cart to Gamescart
- updated the homepage image
- added Footer
- changed relative paths to absolute

---

Screenshot (917)
Screenshot (918)
Screenshot (919)

### CODE REVIEW

File: src/pages/authentication/Login.jsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
-import { Navbar } from '../../components/navbar/Navbar.jsx'
+import { Navbar } from 'components/navbar/Navbar.jsx'
+import { Footer } from 'components/footer/Footer.jsx'
```
Comment (SanketDhabarde) [inline src/pages/authentication/Login.jsx]: You can use barrel export so that you don't have to import again and again it will be done in single line

**Review by SanketDhabarde — APPROVED** [review]
- Code structure looks good
- like the use of grid-area
Your products page title has typo, change it
Que: Just for curiosity, why are you changing relative to an absolute path?

File: src/pages/authentication/Login.jsx (hunk 1 above)
Comment (Rohitprasad83) [reply src/pages/authentication/Login.jsx]: Sure, will do it.
The imports looks good using absolute path, much more readable

**Review by SomeEarth86 — COMMENTED** [review]
Changes look good.

**Pull request merged**